### PR TITLE
Fix SLE15 tests

### PR DESCRIPTION
**Pull request by svet-se**
#### Description:

- _Override the crontabs package for sle15, and install the python-rpm in the test suite for sle15 Dockerfile_

#### Rationale:

- _Add some fixes so the Bash and Ansible tests can be executed without errors_

### CODE REVIEW

File: tests/data/product_stability/alinux2.yml [hunk 1]
```diff
@@ -9,10 +9,10 @@ benchmark_id: ALINUX-2
...
 chrony_conf_path: /etc/chrony.conf
 cpes:
-- alinux2:
-    check_id: installed_OS_is_alinux2
-    name: cpe:/o:alinux:alibaba_cloud_linux:2
-    title: Alibaba Cloud Linux 2
+  - alinux2:
+      check_id: installed_OS_is_alinux2
+      name: cpe:/o:alinux:alibaba_cloud_linux:2
+      title: Alibaba Cloud Linux 2
```
Comment (jan-cerny) [inline tests/data/product_stability/alinux2.yml:15]: @svet-se  What is the reason of these whitespace-only changes?

Comment (svet-se) [reply tests/data/product_stability/alinux2.yml:15]: @jan-cerny it was generated by running "python ./tests/test_product_stability.py --update-reference-data ./products ./tests/data/product_stability" to update the test data

Comment (ggbecker) [reply tests/data/product_stability/alinux2.yml:15]: Sounds like we either have to update the tool to generate files that do not disrupt existing files in terms of indentation or only the meaningful changes are kept in this PR (removing all unnecessary white spaces).

Comment (svet-se) [reply tests/data/product_stability/alinux2.yml:15]: It looks to me like a fix for test_product_stability.py, but for some reason reason the files with the new indentation are not regenerated. IMHO this indentation should be OK. @ggbecker please tell me your thoughts about this. There is also an option to add a parameter for a particular product, i.e. to regenerate the test files only for SLE in this case, not for all products in the directory (but in the particular case we caught 2 products without test files).

Comment (teacup-on-rockingchair) [reply tests/data/product_stability/alinux2.yml:15]: I can confirm that when I run on my setup `python ./tests/test_product_stability.py --update-reference-data ./products ./tests/data/product_stability` I don't get the changes in the indentation so seems like the yaml generator , used in the test_product_stability tool has some environment dependency

Comment (svet-se) [reply tests/data/product_stability/alinux2.yml]: I've reverted the latest commit and just updated the sle15 test data

Comment (svet-se) [reply tests/data/product_stability/alinux2.yml]: @teacup-on-rockingchair it seems that the issue is related to the Python version, I could confirm that with 3.11 there is no issue and the format is correct.

**Review by teacup-on-rockingchair — APPROVED** [review]
:+1:

**Pull request merged**